### PR TITLE
Readme.md

### PR DESCRIPTION
**Pull request by atareshawty**
Two things:

- Updated Readme to provide some explanation of what the code is doing
- Renamed `perceptron.py` to `algorithms.py`. The new file name is a little more descriptive of what it's actually doing

Let me know what you all think!

### CODE REVIEW

**Review by sinasab — COMMENTED** [review]
We'll probably want a short "to run" section. Everything else looks good 👍

**Comment by atareshawty** [comment]
Can do!

**Comment by atareshawty** [comment]
@sinasabet81 new addition to readme is up 🚀

File: README.md [hunk 1]
```diff
@@ -24,6 +24,57 @@ is to average the coefficients produced for each audio file. That way each audio
...
+  - [python_speach_features](https://github.com/jameslyons/python_speech_features)
+  
+- Make sure the data is in this structure:
+
+      ```
+      +-- data/
+      |   +-- trumpet/
+      |   |   +-- Trumpet.vib.ff.A3.aif
+      |   +-- brass/
+      |   |   +-- Trumper.vib.ff.A3.aif
```
Comment (sinasabet81) [inline README.md]: Is a trumper a thing, or is this a typo?

**Review by sinasab — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (atareshawty) [inline README.md]: lol typo. Good catch

**Pull request merged**